pg_duckdb/create.sql

```diff
@@ -0,0 +1,2 @@
+SELECT duckdb.cache('s3://clickhouse-public-datasets/hits_compatible/hits.parquet', 'parquet');
```

Comment:
Instead of caching the S3 file you should also be able to use a local file in the view like this:

```sql
SELECT * FROM read_parquet('/tmp/hits.parquet') AS (...)
```

Reply:
done. mounting greatly sped things up compared to using cache